Jenkinsfile

```diff
@@ -18,315 +18,326 @@ pipeline {
 		lock resource: 'IDLLock'
 	}
 	stages {
-		stage('Build image') {
+		stage("Build image") {
 			steps {
 				script {
 					dockerImage = docker.build("${DOCKER_IMG}")
 				}
 			}
 		}
-		stage('Push image') {
+		stage("Push image") {
 			steps {
 				script {
 					dockerImage.push("${GIT_DESCRIPTION}")
 				}
 			}
 		}
-		stage('DeployTestStack') {
+		stage("DeployTestStack") {
 			when {
 				anyOf {
-					branch 'PR-*'
+					branch "PR-*"
 					expression { return params.forceEndToEnd }
 				}
 			}
+	        steps {
+	            script {
+                    withKubeConfig([credentialsId: "prod-kube-config"]) {
```

Comment:
If all these steps use the `dev` namespace, you can use the `dev-kube-config` credentials.

Reply:
Fixed.